modules/ols-about-product-coverage.adoc

```diff
@@ -5,7 +5,12 @@
 [id="ols-about-product-coverage_{context}"]
 = About product coverage
 
-{ols-official} generates answers to questions based on the content in the official {ocp-product-title} product documentation. The documentation for the following products is not part of the {ocp-short-name} product documentation; therefore, {ols-short} has limited context for generating answers about these products:
+{ols-official} generates answers to questions based on the content in the official {ocp-product-title} product documentation. 
```

Comment:
do we need the word "official" here? It sounds a little odd to me.

Reply:
done